synapseclient/models/mixins/table_components.py

```diff
@@ -495,13 +495,18 @@ async def store_async(
         client = Synapse.get_client(synapse_client=synapse_client)
 
         if self.include_default_columns:
+            view_type_mask = None
+            if self.view_type_mask:
+                if isinstance(self.view_type_mask, ViewTypeMask):
+                    view_type_mask = self.view_type_mask.value
+                else:
+                    view_type_mask = self.view_type_mask
+
```

Comment:
These changes crept in, I didn't intend for them to show up here, but it's fine that they are.

In some cases users wanting to use a single value here can use the enum, but in the case of `FileView` a user may select 2 or more different masks to use. In that case it's an int instead of the Enum.